util/json_storage.rb

```diff
@@ -0,0 +1,14 @@
+require 'json'
+
+class JSONStorage
+  def self.save_data(class_name, object)
+    File.write("./data/#{class_name}.json", JSON.dump(object))
+  end
+
+  def self.load_data(class_name)
+    return [] unless File.exist?("./data/#{class_name}.json")
+
+    file_data = File.read("./data/#{class_name}.json")
+    JSON.parse(file_data, create_additions: true)
+  end
+end
```

Comment:
- [ ] Kindly make sure that the app functions properly. Currently, when I tried to create an item or display a list of items, I get an error that prevents me from testing the app. (To properly  get the error, please make sure to clone the app so you can get a clean version of it and try to test it)

![image](https://user-images.githubusercontent.com/62290545/220995264-6bf4268e-edfb-466e-94e9-898495b91eec.png)

Reply:
Hello @ShadyShawkat 
Thank for the feedback. But your screenshot is not enougth to understand the issue. From which folder are you running the code?(Your screenshot unfortunately doesn't enable us the see this)

As told in our readme, from the project folder, you run `ruby src/main.rb`   to avoid getting that message. 
Thanks.

Reply:
Hi @amentetteh, I did run the app using `ruby src/main.rb`. As I explained, the error happens when you try to display or save data on a new project. Try to clone the project and try the app again.

Reply:
Fixed. Thanks